wpeverest/everest-forms
# Enhancement -  Upgrade to pro link

### All Submissions:


### Changes proposed in this Pull Request:

This PR is adding the new submenu upgrade to pro.

### How to test the changes in this Pull Request:

1. Goto Everest forms.
2. Check the Upgrade to the pro submenu
3. Click and check -> It is redirecting to the pricing page of the Everest form or not.

### Types of changes:


### Other information:


### Changelog entry

>Enhancement -  Upgrade to pro link.
